util/gdas_init/config

```diff
@@ -91,13 +95,31 @@ else
     echo FATAL ERROR: SCRIPTS DO NOT SUPPORT OLD GFS DATA
     exit 2
   elif [ $yy$mm$dd$hh -lt 2016051000 ]; then
-    gfs_ver=v12
+#   gfs_ver=v12
+    set +x
+    echo FATAL ERROR: V12 DATA NOT SUPPORTED.
+    echo CONTACT A REPOSITORY MANAGER FOR ASSISTANCE.
+    exit 2
   elif [ $yy$mm$dd$hh -lt 2017072000 ]; then
-    gfs_ver=v13
+#   gfs_ver=v13
+    set +x
+    echo FATAL ERROR: V13 DATA NOT SUPPORTED.
+    echo CONTACT A REPOSITORY MANAGER FOR ASSISTANCE.
+    exit 2
   elif [ $yy$mm$dd$hh -lt 2019061200 ]; then
-    gfs_ver=v14
+#   gfs_ver=v14
+    set +x
+    echo FATAL ERROR: V14 DATA NOT SUPPORTED.
+    echo CONTACT A REPOSITORY MANAGER FOR ASSISTANCE.
+    exit 2
   elif [ $yy$mm$dd$hh -lt 2021032100 ]; then
     gfs_ver=v15
+    if [ "$CDUMP" = 'gfs' ];then
+    set +x
+      echo FATAL ERROR: V15 DATA NOT SUPPORTED.
```

Comment:
Suggest putting "" around the echo statements in here.

Reply:
Done at 80a3f3f.